test/components/componentTests.ts

```diff
@@ -794,20 +794,50 @@ describe("Component", () => {
 
   describe("restricting rendering through clipPath", () => {
 
-    let c: Plottable.Component;
+    let clippedComponent: Plottable.Component;
     let svg: d3.Selection<void>;
 
     beforeEach(() => {
-      c = new Plottable.Component();
+      clippedComponent = new Plottable.Component();
+      (<any> clippedComponent)._clipPathEnabled = true;
```

Comment:
I think declaring it in the `Mocks` module is fine.  That said, we should probably more consider about renaming that module since some of them could be normal extensions.